### ISSUE TITLE
[BUG] Core dump on launch

### ISSUE THREAD

**Issue by Anikyte** [itsdinkd/AQM2#78]
**Modpack Version**
v1.4.1

**Minecraft Version**
1.18.2

**Describe the crash or bug**
Just after the game window appears, the game crashes. Nothing appears, the window is fully black, it sits there for about a second, and then closes. 

**Steps to reproduce the behavior:**
Import the pack zip into the FTB app, and launch the game.

**Expected behavior**
The game launches as normal.

**Screenshots**
![Screenshot_20220720_154758](https://user-images.githubusercontent.com/48958412/180069137-ee9e25ab-326f-46b7-a2d8-a059a3b1465d.png)

**Crash log or latest.log.log**
[latest.log](https://pastebin.com/NpXyA2uc)
[console.log](https://pastebin.com/cqYAn456)
[Core Dump](https://pastebin.com/LRb6c6g8)

**System Details**
Neofetch:
`OS: Fedora Linux 35 (KDE Plasma) x86_64`
`Host: X570 I AORUS PRO WIFI -CF`
`Kernel: 5.17.8-200.fc35.x86_64`
`Packages: 2410 (rpm), 20 (flatpak)`
`Shell: zsh 5.8.1`
`DE: Plasma 5.24.4`
`WM: kwin`
`CPU: AMD Ryzen 7 3700X (16) @ 3.600GHz`
`GPU: AMD ATI Radeon RX 5700 XT`
`Memory: 6733MiB / 16001MiB`
Java Version:
`openjdk 17.0.3 2022-04-19`
`OpenJDK Runtime Environment 21.9 (build 17.0.3+7)`
`OpenJDK 64-Bit Server VM 21.9 (build 17.0.3+7, mixed mode, sharing)`
[glxinfo](https://pastebin.com/gFG7VazD)
I'm running the standard AMDGPU drivers. No OpenCL.

**Additional context**
Running through the FTB app. I downloaded the pack zip from curseforge and then imported it manually.
I'm able to run other modpacks as normal.

**Comment by Anikyte** [comment]
[hs_err_pid1111010.log](https://github.com/itsdinkd/AQM2/files/9154188/hs_err_pid1111010.log)
pastebin wont let me upload a core dump ¯\_(ツ)_/¯

**Comment by FederAndInk** [comment]
Please see #72, it's a glfw issue, you need glfw>=3.3.7 and use it.

**Comment by Anikyte** [comment]
Solved it! I had glfw 3.3.7, but the problem was FTB app refuses to use it no matter what you do. I had to import the modpack into  MultiMC and enable native GLFW in settings.
![Screenshot_20220720_212806](https://user-images.githubusercontent.com/48958412/180110210-b6237cd9-ecef-461d-852a-6cc295bbdea5.png)
Also, I had to copy libglfw.so.3 n' friends from /usr/lib64 to /usr/lib.
Finally, if you're on Fedora, glfw doesn't seem to come with libglfw.so so you'll just have to copy one of the other ones to it.
So something like:
`cp /usr/lib64/libglfw.* /usr/lib/`
`cp /usr/lib/libglfw.so.3.3 /usr/lib/libglfw.so`

Not sure who supplies the provided libglfw? The fabric team? Whoever, they need to be notified of this so it can be updated.
